util/admin/admin.go

```diff
@@ -123,9 +123,20 @@ func IsJobRollbackable(job *model.Job) bool {
 	return true
 }
 
-// MayNeedBackfill returns whether the action type may need to backfill the data.
-func MayNeedBackfill(tp model.ActionType) bool {
-	return tp == model.ActionAddIndex || tp == model.ActionAddPrimaryKey || tp == model.ActionModifyColumn
+// JobNeedBackfill checks whether a job need to backfill data.
+func JobNeedBackfill(job *model.Job) bool {
+	switch job.Type {
+	case model.ActionAddIndex, model.ActionAddPrimaryKey:
```

Comment:
Do we need to handle it like `ActionModifyColumn`?

Reply:
Done